src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -4,8 +4,15 @@
     <TargetFrameworks>net35;net40;net45;netstandard2.0</TargetFrameworks>
     <RootNamespace>NUnit.Framework</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+  </ItemGroup>
```

Comment:
What do you think about moving these six new lines out of each csproj into the Directory.Build.Props file?

Reply:
I had them there, then decided we only want to do this for the assemblies that we distribute, but it is cleaner, so I am moving them back. Thanks